### PR TITLE
Add graphs to the website

### PR DESCRIPTION
**Pull request by groud**
[Kooha-2024-04-26-16-36-41.webm](https://github.com/godotengine/godot-benchmarks/assets/6093119/26a57c74-645d-4588-925a-3d2ac94c818e)

### CODE REVIEW

File: web/.gitignore [hunk 1]
```diff
@@ -4,7 +4,11 @@ public/
...
-content/*.md
-!_content/_index.md
+# Generated files
+content/benchmark/*.md
+content/graph/*.md
+data/data.json
+
+# Untracked source files
+src-data/benchmarks/*.md
+src-data/benchmarks/*.json
```
Comment (Calinou) [inline web/.gitignore:14]:
```suggestion
src-data/benchmarks/*.json

```

File: web/src-data/graphs.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "id": "rendering-light-and-meches",
+    "title": "Lights and Meshes",
+    "benchmark-path-prefix": "Rendering/Lights And Meshes"
+  },
+  {
+    "id": "rendering-polygon-sprite-2d",
+    "title": "Polygon Sprite 2d",
+    "benchmark-path-prefix": "Rendering/Polygon Sprite 2d"
+  }
+]
```
Comment (Calinou) [inline web/src-data/graphs.json:77]:
```suggestion
]

```

**Comment by coppolaemilio** [comment]
Looks great! imho I would make the page go full with in desktop. I don't think we need to limit the width of it. I would much rather have more info on the screen than having to scroll more. Still, great work 👏

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by Calinou** [comment]
@groud Could you open a PR to update https://github.com/godotengine/godot-benchmarks/blob/main/web/README.md as to update the build instructions? I can't get it to build locally with the following:

```bash
cd web
mkdir src-data
git clone https://github.com/godotengine/godot-benchmarks-results.git benchmarks
cd ..
./generate-content.py
```

Removing `README.md` within `src-data/benchmarks/` doesn't help either (though ideally, this should not be a required step).

**Comment by groud** [comment]
@Calinou You don't have to create the `src-data` folder and the root, it should be already there in the `web` folder. 
I'll update the README.md anyway.

**Pull request merged**